pkg/giantnetes/utils.go

```diff
@@ -307,18 +305,21 @@ func (u *Utils) getEtcdEndpoint(ctx context.Context, cluster Cluster) (string, e
 				return "", microerror.Maskf(executionFailedError, "error creating CAPI cluster kubernetes client for cluster  %#q with error %#q", cluster.clusterKey.Name, err)
 			}
 
-			err = targetCtrlClient.List(ctx, &nodeList, ctrl.HasLabels{LabelCAPIControlPlaneNode})
+			// Specify the label for etcd pods
+			labelSelector := client.MatchingLabels(map[string]string{EtcdLabelComponent: EtcdLabelValue})
```

Comment:
I remember there were other labels, too. Something like `tier: control-plane`. Would it make sense to also add this to have this part more precise?

Reply:
(added that additional label in https://github.com/giantswarm/etcd-backup-operator/pull/438/commits/1cc5fd16469eda0469b09f3fa5c86a8bfcebf1ee)